code/modules/camera_coverage/datum/controller/camera_coverage.dm

```diff
@@ -61,11 +69,15 @@ var/global/datum/controller/camera_coverage/camera_coverage_controller
 /datum/controller/camera_coverage/proc/update_turfs(list/turf/turfs_to_update)
 	if (!length(turfs_to_update))
 		return
+
+	LAZYLISTINIT(src.turf_update_queue)
 	for(var/turf/T as anything in turfs_to_update)
 		if (global.explosions.exploding || ON_COOLDOWN(T, "camera_coverage_update", CAM_TURF_UPDATE_COOLDOWN))
-			LAZYLISTADDUNIQUE(src.turf_update_queue, turfs_to_update)
+			src.turf_update_queue |= turfs_to_update
 			return
 		T.aiImage?.loc = length(T.camera_coverage_emitters) ? null : T
+	if(!length(src.turf_update_queue))
+		src.turf_update_queue = null
```

Comment:
and why null

Reply:
no need for the list if nothing got into it

Reply:
or actually hm, should i just keep it

Reply:
well if it updates again it gotta remake it, why bother